src/styles/card.css

```diff
@@ -0,0 +1,127 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+  
+.card__price-value,
+.card__price-label,
+.card__title-label,
+.card_rate-label,
+.card__title {
+  margin: 0;
+  cursor: default;
+}
+  
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 199px;
+  align-items: center;
+  margin: 0 16px;
+}
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  padding-top: 32px;
+  padding-bottom: 40px;
+}
+
+.card__title-wrapper {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  padding: 0 16px;
+  line-height: 14px;
+  color: #616070;
+  padding-bottom: 16px;
+}
+
+.card__title {
+  cursor: default;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  padding-bottom: 4px;
+}
+
+.card__rate {
+  display: flex;
+  justify-content: space-between;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  width: 166px;
+  margin: 0 auto;
+  color: #060b35;
+  align-items: end;
+  padding-bottom: 24px;
+}
+
+.card_rate-label {
+  cursor: default;
+  display: block;
+  line-height: 14px;
+}
+
+.card__price {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+  display: flex;
+  cursor: default;
+  justify-content: space-between;
+  align-items: center;
+  width: 166px;
+  padding-bottom: 16px;
+}
+
+.card__price-label {
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__price-value {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__button {
```

Comment:
border should be declared here, because as you can see, block changes its size on hover

Reply:
done, thanks